client/src/app/containers/Game/CountAction.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+
+import Button from '@material-ui/core/Button';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import SendIcon from '@material-ui/icons/Send';
+
+import { ActionInputProps } from './types';
+import { useGame } from './useGame';
+
+interface Props extends ActionInputProps {
+    isCrib: boolean;
+}
+
+const CountAction: React.FunctionComponent<Props> = ({
+    isBlocking,
+    isCrib,
+}) => {
+    const [points, setPoints] = useState(0);
```

Comment:
:mag: Should this be `countPts` and `setCountPts`?

Reply:
Does adding the word count increase clarity for you?
To me, `points` was clear enough - this component is used to count a hand, and we know we need to submit a number of points for that. I'd also say that since count can be a verb, that name might get confusing. But curious to hear your take.

Reply:
I'm under the impression that this `points` state is accessible at any point in the component hierarchy - is that not true? If that's not true, then `points` makes the most sense to me

Reply:
Nope! `useState` sets up state _local to the component it's called in_. It can't be accessed elsewhere unless explicitly passed down through props.